redirect.md

```diff
@@ -36,4 +37,44 @@ Method modification:
   }
 ```
 
-To access the target method's parameters, simply append them to the redirector's parameters. 
+## `@Redirect` Instanceof Mode
+
+Parameters, in order:
+
+ - Object that will be the tested object
+ - The original Class being tested against
+ - Returns either a boolean or Class<?> your choice.
+
+**This conflicts with any other mixin targetting the same method call, field access, or object construction. Handle with care.**
+
+Example mixin:
+```java
+@Redirect(method = "target()Z", at = @At(value = "CONSTANT",args="classValue=net/example/Dummy",shift=Shift.AFTER,ordinal=0))
```

Comment:
Please format the code consistently (spaces before and after "=", spaces after ",", braces on same line, consistent indentation)